### ISSUE TITLE
GET https://cdn.jsdelivr.net/gh/processing/p5.accessibility@v0.1.1/dist/p5.accessibility.js net::ERR_ABORTED 404

### ISSUE THREAD

**Issue by liudonghua123** [processing/p5.js#5696]
### Most appropriate sub-area of p5.js?

- [X] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

### p5.js version

1.4.0/1.4.1

### Web browser and version

102.0.5005.115 (Official Build) (64-bit) (cohort: Stable)

### Operating System

Microsoft Windows [Version 10.0.22000.739]

### Steps:
1. open https://editor.p5js.org/
2. open settings and enable Accessibility
3. open console tab of chrome devtools

The correct url for p5.accessibility should be https://cdn.jsdelivr.net/gh/processing/p5.accessibility@0.1.1/dist/p5-accessibility.js, not https://cdn.jsdelivr.net/gh/processing/p5.accessibility@v0.1.1/dist/p5.accessibility.js .

### Snippet:

![image](https://user-images.githubusercontent.com/2276718/174729684-19c67ea1-a950-418c-828c-2a3eda66c8f5.png)

```js
function setup() {
  createCanvas(400, 400);
}

function draw() {
  background(220);
  ellipse(50,50,80,80);
}

```

**Comment by limzykenneth** [comment]
Hi @liudonghua123, is this a p5.js web editor specific problem? If so can you file this issue at the [web editor's repo](https://github.com/processing/p5.js-web-editor/issues) instead? Thanks!
